cfn/ec2.yaml

```diff
@@ -69,48 +64,49 @@ Parameters:
     ConstraintDescription: must be between 5 and 10000
 Resources:
   # ec2 resources
-  EC2Instance:
-    Type: AWS::EC2::Instance
-    Properties:
-      InstanceType: !Ref 'InstanceType'
-      SecurityGroups: [!Ref 'InstanceSecurityGroup']
-      KeyName: !Ref 'KeyName'
-      ImageId: !Ref 'LatestAmiId'
-  InstanceSecurityGroup:
-    Type: AWS::EC2::SecurityGroup
-    Properties:
-      GroupDescription: Enable SSH access via port 22
-      SecurityGroupIngress:
-        - IpProtocol: tcp
-          FromPort: 22
-          ToPort: 22
-          CidrIp: !Ref 'SSHLocation'
+  # EC2Instance:
+  #   Type: AWS::EC2::Instance
+  #   Properties:
+  #     InstanceType: !Ref 'InstanceType'
+  #     SecurityGroups: [!Ref 'InstanceSecurityGroup']
+  #     KeyName: !Ref 'KeyName'
+  #     ImageId: !Ref 'LatestAmiId'
+  # InstanceSecurityGroup:
+  #   Type: AWS::EC2::SecurityGroup
+  #   Properties:
+  #     GroupDescription: Enable SSH access via port 22
+  #     SecurityGroupIngress:
+  #       - IpProtocol: tcp
+  #         FromPort: 22
+  #         ToPort: 22
+  #         CidrIp: !Ref 'SSHLocation'
   # dynamoDB resources
-  myDynamoDBTable:
+  groupEventsTable:
     Type: AWS::DynamoDB::Table
     Properties:
+      TableName: "GroupEvents"
       AttributeDefinitions:
-        - AttributeName: !Ref 'HashKeyElementName'
-          AttributeType: !Ref 'HashKeyElementType'
+        - AttributeName: "eventID"
+          AttributeType: "S"
       KeySchema:
-        - AttributeName: !Ref 'HashKeyElementName'
+        - AttributeName: "eventID"
           KeyType: HASH
       ProvisionedThroughput:
         ReadCapacityUnits: !Ref 'ReadCapacityUnits'
         WriteCapacityUnits: !Ref 'WriteCapacityUnits'
 Outputs:
-  InstanceId:
-    Description: InstanceId of the newly created EC2 instance
-    Value: !Ref 'EC2Instance'
-  AZ:
-    Description: Availability Zone of the newly created EC2 instance
-    Value: !GetAtt [EC2Instance, AvailabilityZone]
-  PublicDNS:
-    Description: Public DNSName of the newly created EC2 instance
-    Value: !GetAtt [EC2Instance, PublicDnsName]
-  PublicIP:
-    Description: Public IP address of the newly created EC2 instance
-    Value: !GetAtt [EC2Instance, PublicIp]
+  # InstanceId:
+  #   Description: InstanceId of the newly created EC2 instance
+  #   Value: !Ref 'EC2Instance'
+  # AZ:
+  #   Description: Availability Zone of the newly created EC2 instance
+  #   Value: !GetAtt [EC2Instance, AvailabilityZone]
+  # PublicDNS:
```

Comment:
we uhh need this field to know where this host. Its fine to remove this for now and I can fix it when I finish automating CI